### PR TITLE
Implement provider-specific sizing for embeds

### PR DESCRIPTION
**Pull request by tituomin**

### CODE REVIEW

File: components/common/StreamField.tsx [hunk 1]
```diff
@@ -289,19 +294,46 @@ const ResponsiveStyles = styled.div`
     height: 100%;
   }
 
-  .responsive-object-small iframe {
-    aspect-ratio: 16 / 9;
-    max-width: 300px;
+  .responsive-object-small {
```
Comment (woodwoerk) [inline components/common/StreamField.tsx:297]: I'm assuming the `responsive-object` class is applied in addition to the sizing classes i.e. `<iframe class="responsive-object responsive-object-small" ... />`, is that the case?

**Review by woodwoerk — APPROVED** [review]
Looks good! Is this the end of the resizable embeds saga? 😁

File: components/common/StreamField.tsx (hunk 1 above)
Comment (tituomin) [reply components/common/StreamField.tsx:297]: Yes!

Comment (tituomin) [reply components/common/StreamField.tsx:297]: @woodwoerk I actually came up with an improvement myself. Would it be bette if I switched these selectors around:
`  .responsive-object-large {
    &.responsive-object[data-embed-provider='${EmbedProvider.YOUTUBE}']`
    
  There would be less repetition of the data-embed-providers. So I mean the enclosing selector would be the .responsive-object selector and they would be "grouped by" provider.

Comment (tituomin) [reply components/common/StreamField.tsx:297]: I mean:
`.responsive-object[data-embed-provider='${EmbedProvider.YOUTUBE}'] { &.responsive-object-large { [...]`

Comment (woodwoerk) [reply components/common/StreamField.tsx]: Nice! Yeah that's much clearer. Super nitpick: you could avoid the repetition of `aspect-ratio` by placing that outside of the `-<size>` specific classes, but this approach may be more readable anyway!

**Review by woodwoerk — APPROVED** [review]

**Pull request merged**